### ISSUE TITLE
Name must be the same as Project ID when adding GCP projects

### ISSUE THREAD

**Issue by devlinrcg** [Netflix/security_monkey#724]
Hi everyone.

Just to let you know that the Develop branch worked perfectly for AWS and GCP account; the thing is that, if you don't setup the account Name the same as the Project ID, you get the following error (YOUR_ACCOUNT_NAME = any account name):

```
2017-05-11 16:16:13,878 ERROR: Job "run_change_reporter (trigger: interval[0:15:00], next run at: 2017-05-11 16:31:04.907523)" raised an exception [in build/bdist.linux-x86_64/egg/apscheduler/scheduler.py:520]
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/usr/local/src/security_monkey/security_monkey/scheduler.py", line 32, in run_change_reporter
    reporter.run(account, interval)
  File "/usr/local/src/security_monkey/security_monkey/reporter.py", line 60, in run
    monitor.watcher.save()
  File "/usr/local/src/security_monkey/security_monkey/watcher.py", line 483, in save
    item.save(self.datastore)
  File "/usr/local/src/security_monkey/security_monkey/watcher.py", line 632, in save
    ephemeral=ephemeral)
  File "/usr/local/src/security_monkey/security_monkey/datastore.py", line 587, in store
    item = self._get_item(ctype, region, account, name)
  File "/usr/local/src/security_monkey/security_monkey/datastore.py", line 652, in _get_item
    raise Exception("Account with name [{}] not found.".format(account))
Exception: Account with name [YOUR_ACCOUNT_NAME] not found.
```

Information slurp are ok, but no information shown on the web console.
Should be an entry on the Quickstart md, I guess...

Anyway, cheers!

**Comment by mikegrima** [comment]
Taking an initial stab at this... It appears to be looking at the project name that appears in the creds JSON.

Will investigate more and look for a fix...

**Comment by mikegrima** [comment]
Fix in #742.  Will test it out.
